CHANGES/6494.bugfix

```diff
@@ -0,0 +1 @@
+URL credentials with zero-length user now supported, e.g. http://:password@host
```

Comment:
This fragment is actually rendered using an RST markup so let's take advantage of that:
```suggestion
Added support for URL credentials with empty (zero-length) username, e.g. ``http://:password@host`` -- by :user:`shuckc`
```